.chalice/config.json

```diff
@@ -2,6 +2,17 @@
   "version": "2.0",
   "app_name": "terraform-registry",
   "stages": {
+    "local": {
+      "api_gateway_stage": "local",
+      "autogen_policy": true,
+      "environment_variables": {
+        "AWS_ACCESS_KEY_ID": "local_id",
+        "AWS_SECRET_ACCESS_KEY": "local_key",
+        "ZTR_DYNAMODB_URL": "http://backend:8000",
```

Comment:
Even though DynamoDB is running locally, botocore still needs fake AWS credentials.